webknossos-datastore/app/com/scalableminds/webknossos/datastore/controllers/DataSourceController.scala

```diff
@@ -81,51 +83,52 @@ class DataSourceController @Inject()(
+          "resumableChunkNumber" -> number,
+          "resumableChunkSize" -> number,
+          "resumableTotalChunks" -> longNumber,
+          "resumableIdentifier" -> nonEmptyText
+        )).fill(("", "", -1, -1, -1, ""))
 
-    accessTokenService.validateAccess(UserAccessRequest.administrateDataSources) {
-      AllowRemoteOrigin {
-        uploadForm
-          .bindFromRequest(request.body.dataParts)
-          .fold(
-            hasErrors = formWithErrors => Fox.successful(JsonBadRequest(formWithErrors.errors.head.message)),
-            success = {
-              case (name, organization, chunkNumber, chunkSize, totalChunkCount, uploadId) =>
-                val id = DataSourceId(name, organization)
-                val resumableUploadInformation = ResumableUploadInformation(chunkSize, totalChunkCount)
-                for {
-                  _ <- if (!uploadService.isKnownUpload(uploadId))
-                    webKnossosServer.validateDataSourceUpload(id) ?~> "dataSet.upload.validation.failed"
-                  else Fox.successful(())
-                  chunkFile <- request.body.file("file") ?~> "zip.file.notFound"
-                  _ <- uploadService.handleUploadChunk(uploadId,
-                                                       id,
-                                                       resumableUploadInformation,
-                                                       chunkNumber,
-                                                       new File(chunkFile.ref.path.toString))
-                } yield {
-                  Ok
-                }
-            }
-          )
+      accessTokenService.validateAccess(UserAccessRequest.administrateDataSources) {
+        AllowRemoteOrigin {
+          uploadForm
+            .bindFromRequest(request.body.dataParts)
+            .fold(
+              hasErrors = formWithErrors => Fox.successful(JsonBadRequest(formWithErrors.errors.head.message)),
+              success = {
+                case (name, organization, chunkNumber, chunkSize, totalChunkCount, uploadId) =>
+                  val id = DataSourceId(name, organization)
+                  val resumableUploadInformation = ResumableUploadInformation(chunkSize, totalChunkCount)
+                  for {
+                    _ <- if (!uploadService.isKnownUpload(uploadId))
+                      webKnossosServer.validateDataSourceUpload(id) ?~> "dataSet.upload.validation.failed"
+                    else Fox.successful(())
+                    chunkFile <- request.body.file("file") ?~> "zip.file.notFound"
+                    _ <- uploadService.handleUploadChunk(uploadId,
+                                                         id,
+                                                         resumableUploadInformation,
+                                                         chunkNumber,
+                                                         new File(chunkFile.ref.path.toString))
+                  } yield {
```

Comment:
You typically omitted the brackets for a single line `yield`, so I'd suggest changing this here, too.